RetailCoder.VBE/UI/Refactorings/Rename/RenameView.xaml.cs

```diff
@@ -5,6 +5,12 @@ public partial class RenameView
         public RenameView()
         {
             InitializeComponent();
+
+            base.Loaded += delegate
```

Comment:
That line is where I was confused.  Found it on the interwebs lol.  IntellSense told me what `base` is, but no idea what `delegate` is or `+=` does.

I understand everything in the second paragraph, but what an "anonymous" handler is.  I'll probably get the answer to that this holiday weekend as I work though as much C# tutorials as I can handle. :)